cypress/integration/universityCanPaySubscription.spec.js

```diff
@@ -77,15 +77,15 @@ describe("University pay for subscription", () => {
     });
 
     cy.get("#submit-payment-button").click();
-    cy.contains("Payment failed!");
+    cy.contains("Payment failed");
   });
 
   it("Payment fails due to invalid inputs", () => {
     cy.route({
       method: "POST",
       url: "http://localhost:3000/api/v0/subscriptions",
       response: "fixture:unsuccessful_subscription_payment_request.json",
-      status: 402
+      status: 200
```

Comment:
This status message does not align with an unsuccessful response.

Reply:
fixed!